packages/esm-appointments-app/src/appointments-calendar/daily/daily-workload-view.component.tsx

```diff
@@ -2,25 +2,24 @@ import dayjs, { type Dayjs } from 'dayjs';
 import classNames from 'classnames';
 import styles from './daily-workload-module.scss';
 import React from 'react';
-import { navigate, useLayoutType } from '@openmrs/esm-framework';
+import { navigate } from '@openmrs/esm-framework';
 import { useTranslation } from 'react-i18next';
 import { spaBasePath } from '../../constants';
 import { isSameMonth } from '../../helpers';
-import { type CalendarType } from '../../types';
+import { type CalendarType, type DailyAppointmentsCountByService } from '../../types';
 
 interface WeeklyCellProps {
   type: CalendarType;
   dateTime: Dayjs;
   currentDate: Dayjs;
-  events: Array<any>;
+  events: Array<DailyAppointmentsCountByService>;
 }
 
 const DailyWorkloadView: React.FC<WeeklyCellProps> = ({ type, dateTime, currentDate, events }) => {
-  const layout = useLayoutType();
   const currentData = events?.find(
     (event) => dayjs(event.appointmentDate).format('YYYY-MM-DD') === dayjs(dateTime).format('YYYY-MM-DD'),
   );
-  const colorCoding = { HIV: 'red', 'Lab testing': 'purple', Refill: 'blue' };
```

Comment:
There were a couple places where there was coloring based on hard-coded service names... I just removed these, we will need to make this configurable somehow if we want this.